HTTPService/pkg/user/service.go

```diff
@@ -0,0 +1,42 @@
+package user
+
+import (
+	"context"
+
+	ent "github.com/coding-kiko/GoKit-Project-Bootcamp/HTTPService/pkg/entities"
+	"github.com/go-kit/kit/log"
+)
+
+type service struct {
+	repo   Repository
```

Comment:
to entify where place an interface you shol be carrefull ith the call of it 
(the service is in fact and exception about the rule)